rust/ql/lib/codeql/rust/elements/internal/VariableImpl.qll

```diff
@@ -463,8 +462,6 @@ module Impl {
     predicate isCapture() { scopeOfAst(this) != scopeOfAst(v.getPat()) }
 
     override string toString() { result = name }
-
-    override string getAPrimaryQlClass() { result = "VariableAccess" }
```

Comment:
Why can't we have this?

Reply:
The Ast consistency queries complain about classes having multiple values for `getAPrimaryQlClass` such as `PathExpr, VariableAccess` and `FormatTemplateVariable,VariableAccess`.

Reply:
Right; so would have to explicitly add `not this intanceof VariableAccess` to the `getAPrimaryQlClass` implementations of `PathExpr` and `FormatTemplateVariable`. It's not critical, though I would rather that we point users towards the more helpful `VariableAccess` class.

Reply:
Alternatively, if we want to have two primary classes, we can simply add is as an exclusion in the consistency check.

Reply:
Something like this https://github.com/github/codeql/pull/17791/commits/a08b4b737201b36ebc273b2b073913b3d6c29b56 ?

Reply:
Yes 👍